zerver/tests/test_templates.py

```diff
@@ -195,6 +195,7 @@ def get_context(self, **kwargs: Any) -> Dict[str, Any]:
                          "login_time": "9:33am NewYork, NewYork",
                          },
             api_uri_context={},
+            zulip_version='1.7.1+git',
```

Comment:
The str value could be read directly from version.py, so that when a version bump happens, only 1 file needs (the version.py) to be modified.

Reply:
@rht That makes sense. OK, I changed it so it used `ZULIP_VERSION` now. Thanks!